src/config/packager.php

```diff
@@ -0,0 +1,8 @@
+<?php
+return [
+    // Use the relative path to stub file.
+//    'service_provider_stub' => '',
+    'author' => '',
+    'author_email' => '',
+    'author_site' => '',
+];
```

Comment:
Missing the service provider stub location

Reply:
Because of the explantation above, i commented the stub location line, so it will pick the default.